web/packages/teleport/src/lib/tdp/codec.ts

```diff
@@ -14,17 +14,8 @@
  * limitations under the License.
  */
 
-import {
-  TextEncoder as TestTextEncoder,
-  TextDecoder as TestTextDecoder,
-} from 'util';
-
```

Comment:
It's worth mentioning that this worked only incidentally: [`util`](https://www.npmjs.com/package/util) is a module which reimplements Node's util module in pure JS. One of msw deps pulls in util, that's why Vite didn't complain about this when compiling this code.

Incidentally, I ran recently into a similar case when requiring `EventEmitter` from `events` which also exists both as a Node.js module and a package in our project deps.